clusters/app.ci/registry-access/baremetal-qe/baremetal-qe.yaml

```diff
@@ -0,0 +1,63 @@
+# this is the Namespace in which your ServiceAccount will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Automation ServiceAccounts for BareMetal QE
+    openshift.io/display-name: BareMetal QE CI
+  name: baremetal-qe
+---
+# this is the ServiceAccount whose credentials you will use
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: image-puller
+  namespace: baremetal-qe
+---
+# this grants your ServiceAccount rights to pull images
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: baremetal-qe-image-puller-binding
+  # the namespace from which you will pull images
+  namespace: ci
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+  - kind: ServiceAccount
+    namespace: baremetal-qe
+    name: image-puller
+---
+# this adds the admins to the project.
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: baremetal-qe-viewer-binding
+  namespace: baremetal-qe
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: view
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: baremetal-qe-admins
```

Comment:
Should this group be the same as https://github.com/openshift/release/pull/55244/files#diff-5cd3644d90fc7c20af1f0484a257857fb5243d9ba0be27f254c8e1200ff013d9R63 ?